src/tfrnnlmbin/lattice-lmrescore-tf-rnnlm.cc

```diff
@@ -0,0 +1,142 @@
+// tfrnnlmbin/lattice-lmrescore-tf-rnnlm.cc
+
```

Comment:
Yes I added a steps/tfrnnlm/check_tensorflow_installed.sh file which checks 1. TF is installed on the Python side 2. TF related binaries are compiled. It gives instructions of how to install them when the checks fail.